### ISSUE TITLE
CLAMD never blocks viruses (Debian)

### ISSUE THREAD

**Issue by TheLastProject** [serghey-rodin/vesta#205]
Upon following [the ClamAV documentation](http://vestacp.com/docs/#how-to-install-clamav-spamassassin-debian-ubuntu), ClamAV does not block viruses. It does scan and find them, but delivers the mail regardless.

acl_m0 is always no:

```
 1146 using ACL "acl_check_data"
 1146 processing "deny"
 1146 check malware = *
 1146 Malware scan: issuing clamd local-path scan [/var/run/clamav/clamd.ctl]
 1146 waiting for data on socket
 1146 Malware response: /var/spool/exim4/scan/1XLYvk-0000IU-I7/1XLYvk-0000IU-I7.eml: Eicar-Test-Signature FOUND
 1146 Malware found, name "Eicar-Test-Signature"
 1146 Matched regex to malware [.+] [Eicar-Test-Signature]
 1146 check condition = ${if eq{$acl_m0}{yes}{yes}{no}}
 1146                 = no
 1146 deny: condition test failed in ACL "acl_check_data"
```

The problem seems to lie with the [exim4 config](https://github.com/serghey-rodin/vesta/blob/master/install/debian/exim4.conf.template). Specifically, lines 117 to 122:

```
  .ifdef CLAMD
    warn set acl_m0 = no
    warn condition = ${if exists {/etc/exim4/domains/$domain/antivirus}{yes}{no}}
    set acl_m0 = yes
  .endif
```

This code never seems to run. When moving it inside of the acl_check_data: function, `${if exists {/etc/exim4/domains/$domain/antivirus}{yes}{no}}` seems to return no, even if the file exists.

tl;dr: exim4 always thinks antivirus is disabled, even if it is not.
